data/campaigns/Under_the_Burning_Suns/scenarios/06a_In_the_Tunnels_of_Trolls.cfg

```diff
@@ -689,14 +696,18 @@
         name=rogrimir
 
         [unit]
+#ifdef EASY
+            type=Dwarvish Sentinel
+#else
             type=Dwarvish Stalwart
+#endif
             id=Rogrimir
             name= _ "Rogrimir"
             profile=portraits/dwarves/fighter-2.webp
             x=41
             y=4
             side=2
-            hitpoints=20
+            hitpoints={ON_DIFFICULTY 40 30 20}
```

Comment:
What are these meant to mean for the player?

It's a very tightly scripted sequence anyway, so difficulty shouldn't make much different; however it will make it much more likely for a bug to go undetected during playtesting if something about the sequence changes in a way that makes HARD mode overly dependent on RNG.

Reply:
When I played through this scenario, the trolls were sometimes killing Rogrimir before I could save him; giving him more HP is supposed to give the player more time to save him. The HARD mode value is preserved because I assumed that HARD mode players were ok with being overly dependent on RNG

Reply:
comment added in [dd0c083](https://github.com/wesnoth/wesnoth/pull/8421/commits/dd0c08340433d9d125b45e46e014426d46b0fdc3).

Reply:
Open to further negotiation about the exact HP values, as long as it stays a linear progression with more HP on easier difficulties.